roles/openshift-patch-resource/README.md

```diff
@@ -0,0 +1,41 @@
+openshift-patch-resource
+========================
+
+Patch a Kubernetes/Openshift resource
+
+Requirements
+------------
+
+This role requires the `kubernetes` python module
+
+Role Variables
+--------------
+
+| Variable | Description | Required | Defaults |
+| :------- | :---------- | :------- | :------- |
+| api_version | The api_version of the resource | yes ||
+| definition | The patch definition | yes ||
+| kind | The kind of the resource | yes ||
+| merge_type | A list of merge types ("strategic-merge, merge") | no | strategic-merge |
```

Comment:
Can we add a link to the [ansible k8s](https://docs.ansible.com/ansible/latest/modules/k8s_module.html) module in the defaults column (maybe as just a link when clicking `strategic-merge`) to indicate that the default is coming from the actual module (i.e.: not set by this role).